src/Backend/Modules/Locale/Actions/Index.php

```diff
@@ -152,8 +152,8 @@ private function loadDataGrid(): void
                 $dataGrid->setColumnAttributes(
                     $lang,
                     [
-                        'data-id' => '{language: \'' .
-                            $lang . '\',application: \'[application]\',module: \'[module]\',name: \'[name]\',type: \'' .
+                        'data-id' => '{\'language\': \'' .
```

Comment:
Yes, here they need to be quoted because they are parsed as JSON in https://github.com/forkcms/forkcms/pull/2212/files#diff-749997ce139a1cb7af2a8c4e05810a31R350